doc/build/kconfig/tips.rst

```diff
@@ -315,6 +315,16 @@ In summary, here are some recommended practices for ``select``:
 - Select simple helper symbols without prompts and dependencies however much
   you like. They're a great tool for simplifying Kconfig files.
 
+- An exemption are busses like I2C and SPI, and following the same thought
+  process things like MFD as well. Drivers on these busses should use
+  ``select`` to allow the automatic activation of the necessary bus drivers
+  when devices on the bus are enabled in the devicetree.
```

Comment:
Should this be moved up ?

So that it's located together with the the `Rare exceptions might ...`:
https://github.com/zephyrproject-rtos/zephyr/blob/bf987033b6c54279340e90c751eb14516f3347b5/doc/build/kconfig/tips.rst?plain=1#L307

Reply:
I've wanted to address this, but it looks like I've been too slow.

Reply:
@benediktibk please send a follow up PR with the change